parameters/navigation/global_costmap.yaml

```diff
@@ -29,14 +29,14 @@ recent_obstacles:
     occupied_to_default_time: 2.0
     # observation_sources: base_laser torso_laser top_kinect
     observation_sources: base_laser top_kinect
-    base_laser:  {data_type: LaserScan, sensor_frame: /amigo/base_laser, topic: /amigo/base_laser/scan, observation_persistence: 0.0, inf_is_valid: true, obstacle_range: 10.0, marking: true, clearing: false, ray_trace_range: 10.0}
+    base_laser:  {data_type: LaserScan, sensor_frame: /amigo/base_laser, topic: /amigo/base_laser/scan, observation_persistence: 0.0, inf_is_valid: true, obstacle_range: 1.5, marking: true, clearing: false, ray_trace_range: 1.5}
```

Comment:
I think this might have been added because of navigation issues on the tournament. Would just keep it at `1.5m`, should be fine.